.travis.yml

```diff
@@ -69,6 +69,7 @@ deploy:
   skip_cleanup: true
   script: bash hack/gen-swagger-doc/deploy.sh
   on:
+    tags: true
```

Comment:
it doesn't get executed now for non-tagged commits. I assumed that it will ... now I read doc and it doesn't :-(